src/overview/assets/index.html

```diff
@@ -205,6 +208,9 @@ <h3 class="font-weight-light">Microservices</h3>
             <div>
               <a href="command:java.helper.openUrl?%22https%3A%2F%2Fcode.visualstudio.com%2Fdocs%2Fazure%2Fdocker%22" title="Learn how to work with Docker in VS Code">Docker in VS Code</a>
             </div>
+            <div>
```

Comment:
Please lift  211-213 and 223-225 to a new division `MicroProfile` before `Microservice` after `Spring`.

Reply:
leave it for now as we need to improve the extension first and then bring it out to have its own category.

Reply:
Okay, we will keep this as-is for now.